src/components/MovieDetail.js

```diff
@@ -0,0 +1,139 @@
+import { Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+import { useLocation } from 'react-router-dom';
+import './css/styling.css';
+function MovieDetail() {
+    
+    const location = useLocation();
+    console.log(location);
+    return (
+        <div>
+            <div class="container">
+                <div class="container_wrap">
+
+                    <div class="content">
+                        <div class="movie_top">
+                            <div class="col-md-9 movie_box">
+
+                                {location.pathname == `/home/tt0076759` ? (
```

Comment:
have a mechanism to filter data from the data list instead of having multiple conditions on the UI to handle what to show. Imagine if the data size is huge e.g. there are thousands of movies so will you have a thousand conditions to show data on the UI?